services/galley/src/Galley/API/Internal.hs

```diff
@@ -188,6 +191,19 @@ type IFeatureStatusWithLock f =
     :<|> IFeatureStatusPut f
     :<|> IFeatureStatusLockStatusPut f
 
+type FeatureNoConfigMultiGetBase featureName =
+  Summary
+    (AppendSymbol "Get team feature status in bulk for feature " (KnownTeamFeatureNameSymbol featureName))
+    :> "features-multi-teams"
+    :> KnownTeamFeatureNameSymbol featureName
+    :> ReqBody '[Servant.JSON] TeamFeatureNoConfigMultiRequest
+    :> Post '[Servant.JSON] (TeamFeatureNoConfigMultiResponse featureName)
+
+type IFeatureNoConfigMultiGet f =
```

Comment:
I would prefer a non-generic type here, if there is no proven need for the generic version.